src/zenml/utils/analytics_utils.py

```diff
@@ -136,6 +136,17 @@ class AnalyticsEvent(str, Enum):
     ZENML_SERVER_DEPLOYED = "ZenML server deployed"
     ZENML_SERVER_DESTROYED = "ZenML server destroyed"
 
+    # ZenML Hub events
+    ZENML_HUB_PLUGIN_LIST = "ZenML Hub plugins listed"
+    ZENML_HUB_LOGIN = "ZenML Hub login"
+    ZENML_HUB_LOGIN_GH = "ZenML Hub login via GitHub"
+    ZENML_HUB_LOGOUT = "ZenML Hub logout"
+    ZENML_HUB_PLUGIN_INSTALL = "ZenML Hub plugin installed"
+    ZENML_HUB_PLUGIN_UNINSTALL = "ZenML Hub plugin uninstalled"
+    ZENML_HUB_PLUGIN_CLONE = "ZenML Hub plugin pulled"
+    ZENML_HUB_PLUGIN_SUBMIT = "ZenML Hub plugin pushed"
+    ZENML_HUB_PLUGIN_LOGS = "ZenML Hub plugin logs viewed"
```

Comment:
Same as the others: Why? Does this give us any useful information? It's not like we can get rid of the functionality to view the build logs if they happen serverside, it's mandatory to debug. Tracking stuff like `% of failed builds` which is interesting again should happen serverside where the actual build is happening.